locust/event.py

```diff
@@ -179,3 +203,31 @@ def __init__(self):
         for name, value in self.__annotations__.items():
             if value == EventHook:
                 setattr(self, name, value())
+
+        setattr(
+            self, "request_success", DeprecatedEventHook("request_success event deprecated. Use the request event.")
```

Comment:
no need to use setattr, just do `self.request_success = DeprecatedEventHook("request_success event deprecated. Use the request event.")`